src/components/feature/feature-mods/feature-mods.config.yml

```diff
@@ -0,0 +1,204 @@
+title: Mods
```

Comment:
This is a bit of nitpick, but I think we could use clearer language here. "Variants", "Test Combinations", or "Configuration options" might be clearer.

Reply:
I think maybe we should talk through this part as a group because I'm finding these less clear. 😬

Reply:
That's fine. Let's defer to some later date when we work on the actual public facing documentation. Since this is just internal documentation for now, the conversation shouldn't stop us moving forward with the feature.